<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SRM_DocumentWidget</class>
 <widget class="QWidget" name="SRM_DocumentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Contact</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Reference no.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Supplier</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="sbRefNo">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Subject</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="leSubject"/>
         </item>
         <item row="0" column="3">
          <widget class="RB_IdxLineEdit" name="ileSupplier"/>
         </item>
         <item row="1" column="3">
          <widget class="RB_IdxLineEdit" name="ileContact"/>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Template</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="RB_IdxLineEdit" name="ileTemplate">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="gbDocument">
         <property name="title">
          <string>Document</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <widget class="QToolButton" name="pbAdd">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../../db/src/gui/db.qrc">
                <normaloff>:/images/icons/add.png</normaloff>:/images/icons/add.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="pbDelete">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../../db/src/gui/db.qrc">
                <normaloff>:/images/icons/delete.png</normaloff>:/images/icons/delete.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="pbSave">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../../db/src/gui/db.qrc">
                <normaloff>:/images/icons/database_save.png</normaloff>:/images/icons/database_save.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leFilter">
              <property name="placeholderText">
               <string>Filter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="pbFilterOn">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../../db/src/gui/db.qrc">
                <normaloff>:/images/icons/filter.png</normaloff>:/images/icons/filter.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="pbFilterOff">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../../db/src/gui/db.qrc">
                <normaloff>:/images/icons/filter_empty.png</normaloff>:/images/icons/filter_empty.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Click to select or [F2] to edit</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QTableView" name="tvDocument"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Content</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="RB_TextEdit" name="teContent"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RB_TextEdit</class>
   <extends>QTextEdit</extends>
   <header location="global">rb_textedit.h</header>
  </customwidget>
  <customwidget>
   <class>RB_IdxLineEdit</class>
   <extends>QLineEdit</extends>
   <header>rb_idxlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tvDocument</tabstop>
  <tabstop>pbAdd</tabstop>
  <tabstop>pbDelete</tabstop>
  <tabstop>pbSave</tabstop>
  <tabstop>leFilter</tabstop>
  <tabstop>pbFilterOn</tabstop>
  <tabstop>pbFilterOff</tabstop>
  <tabstop>sbRefNo</tabstop>
  <tabstop>leSubject</tabstop>
  <tabstop>ileSupplier</tabstop>
  <tabstop>ileContact</tabstop>
  <tabstop>ileTemplate</tabstop>
  <tabstop>teContent</tabstop>
 </tabstops>
 <resources>
  <include location="../../../db/src/gui/db.qrc"/>
 </resources>
 <connections/>
</ui>
